packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -356,10 +356,11 @@ curl -v -X POST \
 #### Create User with Password Import Inline Hook
 
 
-Creates a user with a [Password Hook](#password-hook-object) object specifying that a Password Inline Hook should be used to handle password verification. The Password Inline Hook is triggered to handle verification of the end user's password the first time the user tries to log in. Okta calls the Password Inline Hook to check that the password the user supplied is valid. If the password is valid, Okta stores the hash of the password that was provided and can authenticate the user independently from then on. See [Password Import Inline Hook](/docs/reference/password-hook/) for more details.
+Creates a user with a [Password Hook](#password-hook-object) object specifying that a Password Inline Hook should be used to handle password verification.
 
-The new user is able to login immediately after activation with the valid password.
-This flow supports migrating users from another data store in cases where we wish to allow the users to retain their current passwords.
+The Password Inline Hook is triggered to handle verification of the end user's password the first time the user tries to log in, with Okta calling the Password Inline Hook to check that the password the user supplied is valid. If the password is valid, Okta stores the hash of the password that was provided and can authenticate the user independently from then on. See [Password Import Inline Hook](/docs/reference/password-hook/) for more details.
+
+The new user is able to log in immediately after activation with the valid password. This flow supports migrating users from another data store in cases where we wish to allow the users to retain their current passwords.
```

Comment:
... in **use** cases where we **want** to allow ....

Reply:
The new user is able to sign in immediately.....